File: app/src/main/java/de/chrisward/theyworkforyou/model/Office.java
package de.chrisward.theyworkforyou.model;

import android.arch.persistence.room.PrimaryKey;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

class Office {
    @PrimaryKey(autoGenerate = true)
    @Expose
    public int office_id;

    @SerializedName("dept")
    @Expose
    public String dept;

    @SerializedName("position")
    @Expose
    public String position;

    @SerializedName("from_date")
    @Expose
    public String from_date;

    @SerializedName("to_date")
    @Expose
    public String to_date;

    public Office() {}

    public Office(
            String dept,
            String position,
            String from_date,
            String to_date
    ) {
        this.dept = dept;
        this.position = position;
        this.from_date = from_date;
        this.to_date = to_date;
    }
}

File: app/src/main/java/de/chrisward/theyworkforyou/model/Office.kt
package de.chrisward.theyworkforyou.model

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

@Entity(tableName = "offices")
data class Office (
        @PrimaryKey(autoGenerate = true) @Expose var office_id: Int,
        @SerializedName("dept") @Expose var dept: String,
        @SerializedName("position") @Expose var position: String,
        @SerializedName("from_date") @Expose var from_date: String,
        @SerializedName("to_date") @Expose var to_date: String
)
